src/analysis/properties.rs

```diff
@@ -226,7 +226,7 @@ fn analyze_property(
         None
     };
 
-    if !generate_trait && (writable || readable) {
+    if !generate_trait && (writable || readable || notifable) {
```

Comment:
"notifable" -> Not very english haha

Reply:
Yeah, missing a letter :) Should I fix in this PR or some other time?

Reply:
Ah just did

Reply:
Hehe.